### ISSUE TITLE
Accessibility: Keyboard trap

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52087]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

When a filter by attribute dropdown is at the top of the shop page, the dropdown cannot be tabbed past in Firefox Browser, creating a keyboard trap. Focus is returned to the top of the page and the user cannot move forward. This was observed in Firefox but not in Chrome.

Video: https://app.screencastify.com/v3/watch/1YN5z9TQjeP93aF7qADo

### Component or Page: 

Component: Filter by attribute block

### Name affected element:

Filter by attribute block without the Apply button in Firefox

### Relevant code snippets:

n/a

### Recommended fix:

Ensure that there are no keyboard traps in all browsers.

**Comment by Manussakis** [comment]
The video attached to the PR description is no longer available.
I was not able to reproduce the issue.
Please check the video below:

https://github.com/user-attachments/assets/151eae08-5d1a-409c-8d5e-dcb2d9555347

cc: @vikrampm1

**Comment by vikrampm1** [comment]
Closing this issue as we are unable to reproduce.

**Comment by amberhinds** [comment]
@vikrampm1, this is the one that I am still getting. Here's a video showing the behavior: 

https://github.com/user-attachments/assets/56d7b570-6352-481e-98ce-89539dc93a5f

This only happens in Firefox. When I previously recorded the original video, I was in macOS 14 (Sanoma). I just replicated it again in Firefox,  macOS Sequoia 15.4.1. You need to have VoiceOver turned on to see the issue.

**Comment by Manussakis** [comment]
@Aljullu This issue is related to the Filter Blocks, and I remember you said that, eventually, [these blocks would be replaced](https://github.com/woocommerce/woocommerce/pull/53610#issuecomment-2553284625).
Please let me know if I should invest time in this fix.

Thanks!

cc: @vikrampm1

**Comment by Aljullu** [comment]
Right, I think the new product blocks don't have a dropdown style. cc'ing @dinhtungdu who has more context, just in case I'm saying something wrong.

**Comment by amberhinds** [comment]
I can confirm that the new filter blocks do not have a dropdown style, and there are no keyboard traps with the new filter blocks in Firefox.
